CONTRIBUTING.md

```diff
@@ -60,10 +60,16 @@ When requesting a **New rule**, please create an issue of the **New rule** type.
 
 First of all, **sorry about that!** We want you to have a great time with Detection Rules.
 
-We'll tag issues and pull requests with the target release. If a rule is blocked by a feature, we'll add a label to reflect that. With all of the issues, we need to prioritize according to impact and difficulty, so some issues can be neglected while we work on more pressing issues.
+We'll tag issues and pull requests with the target release if applicable. If a rule is blocked by a feature, we'll add a label to reflect that. With all of the issues, we need to prioritize according to impact and difficulty, so some issues can be neglected while we work on more pressing issues.
 
 Of course, feel free to bump your issues if you think they've been neglected for a prolonged period.
 
+Issues and pull requests (not labeled `security`, `backlog`, `Rule: New`) will be marked as `stale` after 60 days of inactivity. After 7 more days of incactivity, they will be closed automatically.
```

Comment:
```suggestion
Issues and pull requests will be marked as `stale` after 60 days of inactivity. After 7 more days of incactivity, they will be closed automatically.
```

No need to list exceptions, since they will probably drift. I think the next two paragraphs can be deleted, unless they are standard in other repos. We already have the disclaimer before this paragraph. You can just swap them.

I think these notes err a bit on the side of overly polite and a little excessive. I think it's enough that we already encourage contributors to feel free to keep the conversation going.

Short and sweet is good here